draft-ietf-radext-radiusdtls-bis.md

```diff
@@ -236,31 +236,31 @@ RADIUS/(D)TLS client implementations SHOULD implement this model, but MUST imple
-* RADIUS/(D)TLS servers validate the certificate of the RADIUS/(D)TLS client against a local database of acceptable clients.
-  The database may enumerate acceptable clients either by IP address or by a name component in the certificate
-  * For clients configured by DNS name, the configured name is matched against the presented names from the subjectAltName:DNS extension; if no such exist, against the presented CN component in the certificate subject.
-  * For clients configured by their source IP address, the configured IP address is matched against the presented addresses in the subjectAltName:iPAddr extension; if no such exist, against the presented CN component of the certificate subject.
-    For clients configured by IP range, the certificate MUST be valid for the IP address the client is currently using.
-  * It is possible for a RADIUS/(D)TLS server to not require additional name checks for incoming RADIUS/(D)TLS clients, i.e. if the client used dynamic lookup.
-    In this case, the certificate is accepted immediately after the {{RFC5280}} trust chain checks.
-    This MUST NOT be used outside of trusted network environments or without additional certificate attribute checks in place.
-* Implementations MAY allow a configuration of a set of additional properties of the certificate to check for a peer's authorization to communicate (e.g. a set of allowed values in subjectAltName:URI or a set of allowed X.509v3 Certificate Policies).
-* When the configured trust base changes (e.g., removal of a CA from the list of trusted CAs; issuance of a new CRL for a given CA), implementations SHOULD renegotiate the TLS session to reassess the connecting peer's continued authorization.[^may-should-trustbase]{:jf}
+* When the configured trust base changes (e.g., removal of a CA from the list of trusted CAs; issuance of a new CRL for a given CA), implementations SHOULD reassess all connected peer's continued authorization. Note that TLS 1.3 no longer supports renegotiation to fulfill this requirement. [^may-should-trustbase]{:jf}
 
 [^may-should-trustbase]: Open discussion: RFC6614 says "may" here. I think this should be a "should". There are some discussions to change this to "must". Input from TLS/UTA experts is appreciated.
 
+RADIUS/(D)TLS clients and server MUST follow {{!RFC9525}} when validating peer identities. Specific details are provided below:
+* The Common Name RDN MUST NOT be used to identify peers
+* Certifcates MAY use wildcards in the identifiers of DNS names and realm names, but only as the complete, left-most label.
+* RADIUS/(D)TLS clients validate the servers identity to match their local configuration, accepting the identity on the first match:
+  - If the expected RADIUS/(D)TLS server is associated with a specific NAI realm, e.g. by dynamic discovery {{!RFC7585}} or static configuration, that realm is matched against the presented realm names in the subjectAltName:naiRealm extension.
```

Comment:
As per Russ' comments in his [secdir early review](https://mailarchive.ietf.org/arch/msg/radext/LMeW527YwDyovkhP-_2VwmAT_Fk) this needs to be adjusted.

I've tried to change the wording here as one example, I'm not sure if this is correct.
We should probably get confirmation from folks that this is in fact correct to say.

```suggestion
  - If the expected RADIUS/(D)TLS server is associated with a specific NAI realm, e.g. by dynamic discovery {{!RFC7585}} or static configuration, that realm is matched against any naiRealm label contained in the subjectAltName extension.
```

Reply:
Thanks for the heads-up. I will try to incorporate Russ' feedback. Have to check RFC9525 again, maybe it already suggests the correct wording.